wallet-new/src/Cardano/Wallet/Kernel/Actions.hs

```diff
@@ -2,8 +2,8 @@
 module Cardano.Wallet.Kernel.Actions
     ( WalletAction(..)
     , WalletActionInterp(..)
-    , forkWalletWorker
-    , walletWorker
+    , withWalletWorker
+    , Err_WalletWorkerExpired(..)
```

Comment:
@k0001 We don't use this style for type names -- we don't use snake casing but standard camel case -- let's rename this `WalletWorkerExpiredError`.